### ISSUE TITLE
ORI Loss

### ISSUE THREAD

**Issue by Moattakhan** [pedropro/UrsoNet#5]
How to minimize ORI loss. 38 epochs run and still the result of orientation not good..

**Comment by zxx009** [comment]
你好解决了吗，我的ORI损失也是很难降下来

**Comment by Moattakhan** [comment]
Still Better but not good,, i increased the epochs and changed the loss
function.

**Comment by Moattakhan** [comment]
Still Better but not good,, i increased the epochs and changed the loss function

**Comment by zxx009** [comment]
Thank you very much for your reply, in your question you mentioned the change in the loss function can you tell me exactly how to modify it? Of course I will try to increase the training epoch
